go.mod

```diff
@@ -18,7 +18,7 @@ require (
 	github.com/ethereum/go-ethereum v1.9.10
 	github.com/gogo/protobuf v1.3.1
 	github.com/ipfs/go-log v0.0.1
-	github.com/keep-network/keep-common v0.1.1-0.20200228111656-16dd370ca897 // TODO: update to released version
+	github.com/keep-network/keep-common v0.1.1-0.20200330215727-8ca95ff9de56 // TODO: update to released version
 	github.com/keep-network/keep-core v0.10.0
```

Comment:
We will need to update `keep-core` dependency version as soon as it's released.

Reply:
Hope to tag `keep-core` and `keep-common` end of this week.

Reply:
Can this be updated already?

Reply:
I'd prefer to leave it as-is for now and change to RC version end of this week.